### PR TITLE
Add video tutorials to Bascis/Accounts

### LINKED ISSUE #4037: Polish Learn/Basic/Accounts Pages
Minor edits, reorg, link to new video tutorials.

### PR DESCRIPTION
**Pull request by filippoweb3**
This PR is focused on adding recently-created tutorials to the Account Pages of the wiki. Some minor reorg might be possible during the process. The idea is to make the user see that there is a video tutorial before reading the wiki, as this might ease the learning. If the user wants to read, he can return to the wiki.

### CODE REVIEW

**Comment by filippoweb3** [comment]
@alfarok I created this PR to reorganize the Accounts pages by adding the new video tutorials. Let me know what you think.

File: docs/learn/learn-account-generation.md [hunk 1]
```diff
@@ -101,12 +103,13 @@ similar functionality.
...
 secure than using Parity Signer or other air-gapped approaches.
 
-:::info For the tutorial on how to create an account using Polkadot Extension, visit
-[this support article](https://support.polkadot.network/support/solutions/articles/65000098878-how-to-create-a-dot-account).
+### Reset Password using the Browser Extension
 
-:::
+:::info
+
+See [**this video tutorial**](https://www.youtube.com/watch?v=DNU0p5G0Gqc) to learn how to change the password for an account that has been created on the Polkadot-JS browser extension (i.e. an injected account).
```
Comment (profradha) [inline docs/learn/learn-account-generation.md]: We can also specify the timestamp for the videos that take the viewer directly to the specific chapter in the video, especially if the topic appears later in the video.

image


```suggestion
See [**this video tutorial**](https://www.youtube.com/watch?v=DNU0p5G0Gqc&t=280s) to learn how to change the password for an account that has been created on the Polkadot-JS browser extension (i.e. an injected account).
```

File: docs/learn/learn-account-generation.md [hunk 2]
```diff
@@ -198,6 +210,12 @@ account's password".
 
 :::info
 
+See [**this video tutorial**](https://www.youtube.com/watch?v=DNU0p5G0Gqc) to learn how to change the password for an account that has been created on the Polkadot-JS UI (i.e. a non-injected account).
```
Comment (profradha) [inline docs/learn/learn-account-generation.md]:
```suggestion
See [**this video tutorial**](https://youtu.be/DNU0p5G0Gqc?t=261) to learn how to change the password for an account that has been created on the Polkadot-JS UI (i.e. a non-injected account).
```

File: docs/learn/learn-account-generation.md [hunk 3]
```diff
@@ -207,10 +225,15 @@ password of such account.
 
 ## Parity Signer
 
+:::info
+
+For guidelines about how to create an account using Parity Signer, see [**this video tutorial**](https://youtu.be/hgv1R9mPEXw) and visit [**this support article**](https://support.polkadot.network/support/solutions/articles/65000180512-how-to-create-an-account-in-parity-signer).
```
Comment (profradha) [inline docs/learn/learn-account-generation.md]:
```suggestion
For guidelines about how to create an account using Parity Signer, see [**this video tutorial**](https://youtu.be/hgv1R9mPEXw?t=120) and visit [**this support article**](https://support.polkadot.network/support/solutions/articles/65000180512-how-to-create-an-account-in-parity-signer).
```

File: docs/learn/learn-account-restore.md [hunk 4]
```diff
@@ -7,18 +7,42 @@ keywords: [account, restore account, polkadot account, polkadotjs]
...
+
+See [**this video tutorial**](https://youtu.be/9ohp8k4Hz8c) and [**this support page**](https://support.polkadot.network/support/solutions/articles/65000169952-how-to-restore-your-account-in-the-polkadot-extension) to learn how to restore your account on the Polkadot-JS UI.
+
+:::
+
+## Restore Account on Parity Signer
+
+:::info
+
+See [**this video tutorial**](https://youtu.be/hgv1R9mPEXw) and [**this support page**](https://support.polkadot.network/support/solutions/articles/65000167901-how-to-restore-an-account-in-parity-signer) to learn how to restore your account on the Polkadot-JS UI.
```
Comment (profradha) [inline docs/learn/learn-account-restore.md]:
```suggestion
See [**this video tutorial**](https://youtu.be/hgv1R9mPEXw?t=407) and [**this support page**](https://support.polkadot.network/support/solutions/articles/65000167901-how-to-restore-an-account-in-parity-signer) to learn how to restore your account on the Polkadot-JS UI.
```

**Review by DrW3RK — APPROVED** [review]

**Comment by DrW3RK** [comment]
Looks good to me. Added the timestamps to further improve user experience.

File: docs/learn/learn-accounts.md [hunk 5]
```diff
@@ -108,21 +114,29 @@ but gets put back when it has the existential deposit.
 
 The [Identities pallet](https://github.com/paritytech/substrate/tree/master/frame/identity) built
 into Polkadot allows users to attach on-chain metadata to their accounts. Independent registrars can
-verify this metadata to provide trustworthiness. To learn more about how to set or release an
-identity, how to define sub-accounts, or how to become a registrar, please read
-[this guide](learn-identity.md).
+verify this metadata to provide trustworthiness.
+
+:::info To learn more about how to set or release an identity, how to define sub-accounts, or how to become a registrar, please read [this guide](learn-identity.md).
```
Comment (profradha) [inline docs/learn/learn-accounts.md]:
```suggestion
:::info 

To learn more about how to set or release an identity, how to define sub-accounts, or how to become a registrar, please read [this guide](learn-identity.md).
```

File: docs/learn/learn-accounts.md [hunk 6]
```diff
@@ -108,21 +114,29 @@ but gets put back when it has the existential deposit.
...
+:::
 
 ## Proxy Accounts
 
 Polkadot comes with a generalized proxy account system that allows users to keep keys in cold
-storage while proxies act on their behalf with restricted (or unrestricted) functionality. See the
-[proxies](learn-proxies.md) page for more information.
+storage while proxies act on their behalf with restricted (or unrestricted) functionality.
+
+:::info See the [proxies](learn-proxies.md) page for more information about proxies.
```
Comment (profradha) [inline docs/learn/learn-accounts.md]:
```suggestion
:::info 

See the [proxies](learn-proxies.md) page for more information about proxies.
```

File: docs/learn/learn-accounts.md [hunk 7]
```diff
@@ -108,21 +114,29 @@ but gets put back when it has the existential deposit.
...
+:::
 
 ## Multi-signature Accounts
 
 Multi-signature accounts are accounts created from several standard accounts (or even other
-multi-sig accounts). For a full explanation, please see the
-[Multi-signature accounts page](./learn-account-multisig.md).
+multi-sig accounts).
+
+:::info  See the [Multi-signature accounts page](./learn-account-multisig.md) for a full explanation about multi-signature accounts, including their use-cases.
```
Comment (profradha) [inline docs/learn/learn-accounts.md]:
```suggestion
:::info  

See the [Multi-signature accounts page](./learn-account-multisig.md) for a full explanation about multi-signature accounts, including their use-cases.
```

**Review by DrW3RK — COMMENTED** [review]
Moving the content inside the info windows. Right now, the content is rendered fully CAPS differing from previous info windows

image

File: docs/learn/learn-balance-transfers.md [hunk 8]
```diff
@@ -15,52 +15,46 @@ transferring a balance, we will begin by using
...
-![Companion transfer](../assets/accounts/companion-transfer.png)
+:::
 
-Scan the QR code using Parity Signer. You will see a QR code consisting of the signed extrinsic for
-this transaction. Click on the **next to signing** button in the pop-up window, as shown above. You
-can then show the QR code on your phone within the red square window.
+## Keep-Alive Checks
 
-![Companion transfer](../assets/accounts/companion-transfer2.png)
+:::info See [**this video tutorial**](https://youtu.be/Wg0pH05CC9Y) and [**this support page**](https://support.polkadot.network/support/solutions/articles/65000168651-what-is-the-existential-deposit-) to learn about keep-alive checks and existential deposit.
```
Comment (profradha) [inline docs/learn/learn-balance-transfers.md]:
```suggestion
:::info 

See [**this video tutorial**](https://youtu.be/Wg0pH05CC9Y) and [**this support page**](https://support.polkadot.network/support/solutions/articles/65000168651-what-is-the-existential-deposit-) to learn about keep-alive checks and existential deposit.
```

File: docs/learn/learn-account-generation.md [hunk 9]
```diff
@@ -22,15 +22,11 @@ security trade-offs. Please review this page carefully before making your accoun
 understand the risks of the account generation method you choose and how to properly mitigate them
 in order to keep your funds safe.<br/><br/>
 
-:::info
-
-Visit
-[this support article](https://support.polkadot.network/support/solutions/articles/65000181874-how-to-store-your-mnemonic-phrase-and-backup-file-safely)
-for more information about key security.
+:::info Visit [this support article](https://support.polkadot.network/support/solutions/articles/65000181874-how-to-store-your-mnemonic-phrase-and-backup-file-safely) for more information about key security.
```
Comment (alfarok) [inline docs/learn/learn-account-generation.md]:
```suggestion
:::info 

Visit [this support article](https://support.polkadot.network/support/solutions/articles/65000181874-how-to-store-your-mnemonic-phrase-and-backup-file-safely) for more information about key security.
```

File: docs/learn/learn-balance-transfers.md [hunk 10]
```diff
@@ -15,52 +15,46 @@ transferring a balance, we will begin by using
...
+[**Parity Signer**](https://www.parity.io/technologies/signer/) for signing transactions. We do not
 provide support for third party applications.
 
 :::
 
-:::info Most of the content on this page has been moved to several support pages. See the links
-below.
+## Sending Funds using UI, Extension, Parity Signer & Ledger
+
+:::info See the video tutorial below to learn how to send funds using the supported tools.
```
Comment (alfarok) [inline docs/learn/learn-balance-transfers.md]:
```suggestion
:::info

See the video tutorial below to learn how to send funds using the supported tools.
```

File: docs/learn/learn-balance-transfers.md [hunk 11]
```diff
@@ -15,52 +15,46 @@ transferring a balance, we will begin by using
...
-  - [Signing with the browser extension](https://support.polkadot.network/support/solutions/articles/65000181989)
-  - [Signing with ledger](https://support.polkadot.network/support/solutions/articles/65000181994)
-  - [Signing with parity signer](https://support.polkadot.network/support/solutions/articles/65000182000)
 
-## Signing Transactions with Parity Signer
+<iframe width="560" height="315" src="https://youtube.com/embed/gbvrHzr4EDY" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe><br/><br/>
 
-:::info This information is complementary to the
-[dedicated support page](https://support.polkadot.network/support/solutions/articles/65000182000).
+:::info Transfer of Funds
```
Comment (alfarok) [inline docs/learn/learn-balance-transfers.md:38]:
```suggestion
:::info

Transfer of Funds
```

File: docs/learn/learn-balance-transfers.md [hunk 12]
```diff
@@ -15,52 +15,46 @@ transferring a balance, we will begin by using
...
 
-If you directly import your Parity Signer account to Polkadot-JS UI, you will see a pop-up window
-with a QR code and a camera window, as shown below. Open the QR scanner on Parity Signer and scan
-the QR code. You may be prompted to enter your PIN to sign the transaction. The signed extrinsic
-will then be available as a QR code that needs to be shown in front of your computer's camera (as
-shown in the red square window shown to the right in the screenshot below).
+:::
 
-![Companion transfer](../assets/accounts/signer-transaction.png)
+:::info Signing Transactions
```
Comment (alfarok) [inline docs/learn/learn-balance-transfers.md:45]:
```suggestion
:::info

Signing Transactions
```

**Review by alfarok — APPROVED** [review]
@filippoweb3 Great additions!  I just added a few more instances of the same formatting issue @DrW3RK has mentioned.  Let me know if there is an issue with `prettier` that is causing any unwanted auto-formatting when committing.  I think as long as there is a newline between the opening `:::info` and closing `:::` tags it shouldn't take any action.

**Pull request merged**